pkgs/tools/system/pciutils/default.nix

```diff
@@ -11,7 +11,7 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [ pkgconfig ];
   buildInputs = [ zlib kmod which ];
 
-  makeFlags = "SHARED=yes PREFIX=\${out}";
+  makeFlags = [ "SHARED=yes" "PREFIX=\${out}" "STRIP=" "HOST=${stdenv.hostPlatform.system}" "CROSS_COMPILE=${stdenv.cc.targetPrefix}" ];
```

Comment:
Uh I don't think this alone is enough? @dtzWill and I ran into problems before with `make` running `./lib/Configure` without any arguments.

Reply:
What kind of problems? It builds fine for me and @dtzWill tested it as well.

Reply:
I suppose we tried fewer arguments there; maybe these are sufficient.

Reply:
Last thing, normally `HOST=${stdenv.hostPlatform.config}` would be better. Are you sure the shorter nix-style names are superior?

Reply:
The `HOST` variable is parsed using custom code in the configure script (see [here](/pciutils/pciutils/blob/master/lib/configure#L18)), so it does not follow a standard format (such as the LLVM target triple). When it is not manually specified, it is pieced together from `uname -s`, `uname -r` and a number of other commands depending on the OS. It just so happens that it matches the value of `stdenv.hostPlatform.config` for most systems. The only exception I can see is that when `stdenv.hostPlatform.config == "i686-linux"`, the value of `HOST` should be `i386-linux` (see [line 28](/pciutils/pciutils/blob/master/lib/configure#L28)), so this may break i686 builds, but I don't have a way to test this.

Reply:
Cool, thanks for the diligent research. If you put that in a comment, this is fine with me. We just need a patchShebangs solution then?